mllib/src/main/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluator.scala

```diff
@@ -41,6 +41,10 @@ class MulticlassClassificationEvaluator @Since("1.5.0") (@Since("1.5.0") overrid
   /**
    * param for metric name in evaluation (supports `"f1"` (default), `"precision"`, `"recall"`,
    * `"weightedPrecision"`, `"weightedRecall"`, `"accuracy"`)
+   * Note: `"f1"` here means weighted averaged f1-measure, while the `fMeasure` in
```

Comment:
Also, fMeasure isn't a supported value here? or do I miss it? f1 is a measure. Here again I wonder if it's worth documenting it, since f1 / precision / recall are deprecated.